controllers/topics.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchApiTopics } = require("../models/topics.models");
+
+exports.getApiTopics = (req, res, next) => {
+    fetchApiTopics().then((result) => {
+        res.status(200).send({ topics: result });
```

Comment:
As you are expecting topics back from the database, is result the most semantic variable you could have chosen?

Reply:
Changed the name to topics.